search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -71,15 +76,14 @@ function toggleDisplay(id) {
  * @param {string} countryCode Two letter country code for selected country.
  */
 function displayTopResultsForCurrentTrend(countryCode) {
-  let dataByCountry = getCurrentSearchData().dataByCountry;
   let date = new Date(getCurrentSearchData().timestamp);
   let resultElement =  document.getElementById('search-results-tab');
   resultElement.innerHTML = '';
 
-  let countryData = dataByCountry.filter(data => data.country === countryCode);
+  let countryData = getCurrentSearchData().dataByCountry.filter(data => 
+      data.country === countryCode);
```

Comment:
Best practice is to split at the highest syntactic level, so either after the initial `=` or instead put the `.filter` all on its own line I think.

https://google.github.io/styleguide/jsguide.html#formatting-where-to-break